admin/services/school.service.js

```diff
@@ -52,14 +65,12 @@ const schoolService = {
   },
 
   /**
-   * @typedef editableSchoolDetails
-   * {
-   *   dfeNumber: number,
-   *   estabCode: number,
-   *   leaCode: number,
-   *   name: string,
-   *   urn: number
-   * }
+   * @typedef {object} editableSchoolDetails
+   * @property {any} dfeNumber,
```

Comment:
There are a lot of {any} types here.  Can they not be tightened up a bit?

Reply:
tightened up, along with incoming submission values